src/clj/clj_pdf/core.clj

```diff
@@ -672,13 +672,14 @@
         (swap! *cache* assoc svg-hash compiled)
         compiled))))
 
-(defn- line [{dotted? :dotted, gap :gap} & _]
+(defn- line [{dotted? :dotted, gap :gap, [r g b] :color} & _]
   (let [^LineSeparator lineSeparator (if dotted?
                                        (if gap
                                          (doto (new DottedLineSeparator) (.setGap (float gap)))
                                          (new DottedLineSeparator))
                                        (new LineSeparator))]
-    (doto lineSeparator (.setOffset -5))))
+    (doto lineSeparator (.setLineColor (if (and r g b) (new Color r g b)))
```

Comment:
I don't think you want to set line color at all if the check fails, I'd recommend:

```clojure
(if (and r g b) (.setlineColor lineSeparator (Color. r g b)))
(.setOffset lineSeparator -5)
```